evap/evaluation/templates/confirmation_text_modal.html

```diff
@@ -0,0 +1,46 @@
+{% load evaluation_filters %}
+
+<div class="modal fade" id="{{ modal_id }}" tabindex="-1" role="dialog" aria-labelledby="{{ modal_id }}Label" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="{{ modal_id }}Label">{{ title }}</h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span class="fas fa-times"></span>
+                </button>
+            </div>
+            <form id="{{ modal_id }}Form">
+                <div class="modal-body">
+                    {{ question|safe }}
+                    <br /><br />
```

Comment:
Personally, I'd prefer using just `<br>`. Do we have a policy for this?

Check out https://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br

Reply:
i prefer `<br />`, because i expect every opening tag to be closed. same with, e.g., `<input />`. we're using the self closing version more often (57 vs 15 times) - we should probably make it consistent everywhere.

Reply:
Fair enough. Follow up issue?

Reply:
general cleanup or just br-tags? i could add a commit for just those right here.

Reply:
So far I've only noticed the br tags, might be the same for some of the other void tags though (https://www.w3.org/TR/html5/syntax.html#void-elements)